CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ The main work (all changes without a GitHub username in brackets in the below li
 	## __WORK IN PROGRESS__
 -->
 
+## __WORK IN PROGRESS__
+-   @matter/protocol
```

Comment:
Open a dummy comment ... I send this to the current Controller implementors to request feedback ...

> Do you have any opinion if this is breaking or makes sense? I found it problematic getting the switch button events but then the "currentPosition" was reported as "0" before these events. if you consider breaking I would make it configurable ... else just is different behavior. WDYT?